src/odemis/gui/cont/acquisition.py

```diff
@@ -2272,50 +2274,126 @@ def _on_ac_done(self, future):
 
 
 class FastEMAlignmentController:
-    def __init__(self, tab_data, panel):
+    """
+    Takes care of the calibration button and process in the calibration panel in the FastEM acquisition tab.
+    """
+    def __init__(self, tab_data, tab_panel):
+        """
+        tab_data: (FastEMAcquisitionGUIData) The representation of the microscope GUI.
+        tab_panel: (wx.Frame) The calibration panel, which contains the calibration button,
+                    the gauge and the label of the gauge.
+        """
         self._tab_data = tab_data
-        self._panel = panel
+        self._main_data_model = tab_data.main
+        self._panel = tab_panel
+
+        self.btn_align = tab_panel.btn_align
+        self.gauge_progress = tab_panel.align_gauge_progress  # progress bar
+        self.gauge_label = tab_panel.align_lbl_gauge  # progress bar label
+        self.is_aligned = tab_data.main.is_aligned
+
+        self.gauge_progress.Hide()  # hide progress bar
+        self._panel.Parent.Layout()
+        self.btn_align.Bind(wx.EVT_BUTTON, self.on_align)
+
+        # FIXME: How to differentiate between simulator and real HW?
+        #   The following two calibrations can also be run with the simulator.
+        #   However, the rest not. How make compartible with both cases?
+        # CALIBRATIONS_SIM and CALIBRATIONS as global var somehow?
```

Comment:
If you decide to make this distinction in this module and not in the test cases, to keep this class readable I would add `FastEMAlignmentControllerSimulator` here and not have it mixed in this code.

Reply:
That is indeed an idea. On the other hand all the rest of the code is the same. So I am a bit undecided. Is passing the calibrations in the init an option?

Reply:
that might work, if it's only one or two lines it's also fine to have it here, just try to keep readability in mind

Reply:
@pieleric how would you suggest to solve this?

Reply:
There are two questions.
* How to know whether the current hardware is a simulator or not. I don't have any great idea. I think it should be some sort of flag on the microscope file. 
* In the GUI, I agree it's nice to be able to change just a attribute like `.calibrations` based on whether it's the simulator or not. If that's just a line, indeed, a single `if` is simplest to read. If it's more complex, let's use a subclass (of which .calibrations can be a class attribute).